@xen-orchestra/backups/docs/VM backups/README.md

```diff
@@ -76,12 +78,13 @@ job.start(data: { mode: Mode, reportWhen: ReportWhen })
 │  │  ├─ task.start(message: 'clean')
 │  │  │  ├─ task.warning(message: string)
 │  │  │  └─ task.end
-│  │  │
-│  │  │ // in case of delta backup
+│  │  └─ task.end
+|  ├─ task.start(message: 'clean-vm')
+│  │  ├─ task.warning(message: string)
+│  │  │  // in case of delta backup
```

Comment:
The merge subtask can also occurs before the backup.

IMHO, we can remove it entirely, it's implementation detail at this time.